### ISSUE TITLE
I can't install input-leap in nixos

### ISSUE THREAD

**Issue by fe2-Nyxar** [input-leap/input-leap#1906]
### What happened?

unable to install input-leap on both gnome and plasma 5, running X11.
I tried all methods, nix-shell nix-env , and declaring it.

### Version

v2.0.0

### If applicable, where did you install Input Leap from?

nix store

### What OSes are you seeing the problem on? (Check all that apply)

Linux

### If applicable, are you using Wayland or X11?

X11

### What OS versions are you using?

Nixos

### Relevant log output

```shell
[ 83%] Building CXX object src/lib/platform/CMakeFiles/platform.dir/EiScreen.cpp.o
[ 83%] Building CXX object src/lib/platform/CMakeFiles/platform.dir/PortalInputCapture.cpp.o
[ 84%] Building CXX object src/lib/platform/CMakeFiles/platform.dir/PortalRemoteDesktop.cpp.o
/build/source/src/lib/platform/EiEventQueueBuffer.cpp: In member function 'virtual void inputleap::EiEventQueueBuffer::waitForEvent(double)':
/build/source/src/lib/platform/EiEventQueueBuffer.cpp:99:17: warning: ignoring return value of 'ssize_t read(int, void*, size_t)' declared with attribute 'warn_unused_result' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result-Wunused-result8;;]
   99 |             read(pipe_r_, buf, sizeof(buf)); // discard
      |             ~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/source/src/lib/platform/EiEventQueueBuffer.cpp: In member function 'virtual bool inputleap::EiEventQueueBuffer::addEvent(uint32_t)':
/build/source/src/lib/platform/EiEventQueueBuffer.cpp:141:10: warning: ignoring return value of 'ssize_t write(int, const void*, size_t)' declared with attribute 'warn_unused_result' [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wunused-result-Wunused-result8;;]
  141 |     write(pipe_w_, "!", 1);
      |     ~~~~~^~~~~~~~~~~~~~~~~
/build/source/src/lib/platform/PortalRemoteDesktop.cpp: In member function 'void inputleap::PortalRemoteDesktop::cb_session_started(GObject*, GAsyncResult*)':
/build/source/src/lib/platform/PortalRemoteDesktop.cpp:122:36: error: too few arguments to function 'int xdp_session_connect_to_eis(XdpSession*, GError**)'
  122 |     fd = xdp_session_connect_to_eis(session);
      |          ~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~
In file included from /nix/store/gafrw30lmknz9x9h94fpdbvm6s325cv9-libportal-0.7.1-dev/include/libportal/portal.h:35,
                 from /build/source/src/./lib/platform/PortalRemoteDesktop.h:26,
                 from /build/source/src/lib/platform/PortalRemoteDesktop.cpp:18:
/nix/store/gafrw30lmknz9x9h94fpdbvm6s325cv9-libportal-0.7.1-dev/include/libportal/remote.h:203:11: note: declared here
  203 | int       xdp_session_connect_to_eis    (XdpSession  *session,
      |           ^~~~~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [src/lib/platform/CMakeFiles/platform.dir/build.make:370: src/lib/platform/CMakeFiles/platform.dir/PortalRemoteDesktop.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:667: src/lib/platform/CMakeFiles/platform.dir/all] Error 2
make: *** [Makefile:166: all] Error 2
error: builder for '/nix/store/a3hanpmg44ckw24kicm6yaw5fqayx3bx-input-leap-unstable-2023-05-24.drv' failed with exit code 2;
       last 10 log lines:
       > In file included from /nix/store/gafrw30lmknz9x9h94fpdbvm6s325cv9-libportal-0.7.1-dev/include/libportal/portal.h:35,
       >                  from /build/source/src/./lib/platform/PortalRemoteDesktop.h:26,
       >                  from /build/source/src/lib/platform/PortalRemoteDesktop.cpp:18:
       > /nix/store/gafrw30lmknz9x9h94fpdbvm6s325cv9-libportal-0.7.1-dev/include/libportal/remote.h:203:11: note: declared here
       >   203 | int       xdp_session_connect_to_eis    (XdpSession  *session,
       >       |           ^~~~~~~~~~~~~~~~~~~~~~~~~~
       > make[2]: *** [src/lib/platform/CMakeFiles/platform.dir/build.make:370: src/lib/platform/CMakeFiles/platform.dir/PortalRemoteDesktop.cpp.o] Error 1
       > make[2]: *** Waiting for unfinished jobs....
       > make[1]: *** [CMakeFiles/Makefile2:667: src/lib/platform/CMakeFiles/platform.dir/all] Error 2
       > make: *** [Makefile:166: all] Error 2

```

**Comment by shymega** [comment]
This is because Nixpkgs is missing a dependency in their derivation - I don't think this is a Input Leap issue. Other packaging works fine.

**Comment by shymega** [comment]
Ah, I see - the update to the derivation wasn't backported until recently. This issue should now be resolved, as it's been backported to Nixpkgs 23.11 - thank you, @eclairevoyant!

Not an Input Leap issue, this will remain closed as wontfix.
